From 52c401002cfe3102683ca3b2f25bb734979f61c3 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 3 Aug 2020 14:24:48 +0800
Subject: [PATCH] Test case9-E for Change value of PcdsDynamicHii PCD

Description: Change value of PcdsDynamicHii PCD to  ("gEfiMdeModulePkgTokenSpaceGuid.PcdUse1GPageTable|L"1GPageTable"|gEfiGenericVariableGuid|0x0|FALSE") in TestPkg/TestPkg32X64.dsc.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/TestPkg32X64.dsc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/TestPkg/TestPkg32X64.dsc b/TestPkg/TestPkg32X64.dsc
index 8c881de93b..a3b78884d1 100644
--- a/TestPkg/TestPkg32X64.dsc
+++ b/TestPkg/TestPkg32X64.dsc
@@ -190,7 +190,7 @@ gUefiCpuPkgTokenSpaceGuid.PcdCpuMaxLogicalProcessorNumber|64
 gEfiMdeModulePkgTokenSpaceGuid.PcdFlashNvStorageVariableBase64|0x8000001
 
 [PcdsDynamicHii]
-gEfiMdeModulePkgTokenSpaceGuid.PcdUse1GPageTable|L"1GPageTable"|gEfiGenericVariableGuid|0x0|TRUE
+gEfiMdeModulePkgTokenSpaceGuid.PcdUse1GPageTable|L"1GPageTable"gEfiGenericVariableGuid|0x0|FALSE
 
 [PcdsDynamicExVpd.common.DEFAULT]
- gEfiMdeModulePkgTokenSpaceGuid.PcdNvStoreDefaultValueBuffer|*
\ No newline at end of file
+#gEfiMdeModulePkgTokenSpaceGuid.PcdNvStoreDefaultValueBuffer|*
\ No newline at end of file
-- 
2.27.0.windows.1

